### ISSUE TITLE
Constant vibration when charging

### ISSUE THREAD

**Issue by ajack2001my** [InfiniTimeOrg/InfiniTime#724]
### 

- [X] I searched for similar bug reports and found none was relevant.

### What happened?

While charging my PineTime, the watch sonstantly vibrates

### What should happen instead?

Should not vibrate

### Reproduction steps

Just charge the watch

### Version

InfiniTime 1.6.0

### Companion app

GadgetBridge 0.60.0

**Comment by ildar** [comment]
https://wiki.pine64.org/index.php/PineTime#Chat

**Comment by ajack2001my** [comment]
> 
> Would you make a little experiment? hold your finger for a little while on the watch. Does it stop?

No, just tested. Anything else I can do or test?

**Comment by kennysc** [comment]
I have the same bug on my sude with:
Infinitime 1.7.1
Gadgetbridge 0.62.0

Did anyone found a solution for this ?

**Comment by Riksu9000** [comment]
I just had something like this happen. The watch had been charging for a while, then suddenly it vibrated a few times, and soon after that it started vibrating constantly like the vibration motor was left on, so I quickly reset the watch to get it to stop. It didn't stop by just taking it off the charger. Also I'm not sure it woke up at any point, or maybe I just didn't notice when I hurried to reset it.

**Comment by InFerYes** [comment]
I have this buzzing issue with 1.7.1 where it vibrates on and off. The vibrating stops when I take it off the charger. It's not a constant vibration so I don't know if it's the same issue and warrants it's own thread.

https://www.youtube.com/watch?v=ONSYH0_wEfA

**Comment by ajack2001my** [comment]
This issue and phantom inputs are what's getting to me. Appreciate it if somebody can look into these...

**Comment by Riksu9000** [comment]
Have you cleaned the contacts and tried different chargers? Can you share a video showing the exact behaviour?

**Comment by ajack2001my** [comment]
> I just had something like this happen. The watch had been charging for a while, then suddenly it vibrated a few times, and soon after that it started vibrating constantly like the vibration motor was left on, so I quickly reset the watch to get it to stop. It didn't stop by just taking it off the charger. Also I'm not sure it woke up at any point, or maybe I just didn't notice when I hurried to reset it.
> 
> EDIT: Don't know what firmware I was running unfortunately, so it could've been some PR. Probably just firmware crashed or something

You described it exactly... but stops when i take it off the charger. Contacts are clean...

**Comment by lightweight** [comment]
Happening to me, too. Running 1.7.1. If I put the watch on the charger, it just does sporadic (fairly random) vibrations, constantly. Taking it of the charger stops it instantly.

**Comment by nooploop** [comment]
Have same issue with 1.8.0

**Comment by WPettersson** [comment]
tl;dr -> possibly due to almost-low USB power to charger?

I recently received my first PineTime, and noticed this issue with both the shipped Infinitime and latest version (1.8.0), but not with wasp, which seemed odd. However, I also knew from past experience that the charger was plugged into a USB port that would slowly charge my phone. Initially, this was to compare how fast the watch charges between USB ports on my PC (phone charges faster) and USB ports on a hub (phone charges slower), but I also noticed that the vibrate-while-charging would only ever happen on the hub-ports, never on my PC. I've checked a few times, the watch charges very slowly off the hub and vibrates often, but when plugged into a PC directly it charges fast and does not vibrate.

**Comment by hrmckay** [comment]
This just happened to me this morning.  Version 1.8.0.  I have charged the watch multiple times, always from my PC, and it has not happened before.  I removed the watch from the cradle and the vibration stopped.  I put it back and the vibration started again.  I checked the battery level and it was 46%.  After removing the watch and putting it back with the vibration always recurring, I squeezed the watch (I have a dev kit held together by scotch tape) and put it back again and the vibration did not occur.

Hugh

**Comment by minacode** [comment]
> I suggest making this a maximum priority issue

I guess the best way to handle this is opening a PR.

**Comment by Sn3akyP3t3** [comment]
I think I just encountered this issue.  The cause it seems was the contacts were dirty from bio residue, dirt, sweat, etc.  Cleaning the contacts resolved the spontaneous vibration sessions while charging.

I think for some the vibration would be helpful for accessibility to get that vibration confirmation on charge connection.  I don't have as suggestion for a workaround.  Mitigation suggestions would only reduce the frequency.
